install/src/command.rs

```diff
@@ -786,44 +795,59 @@ pub fn update(config_file: &str) -> Result<bool, String> {
             ExplicitRelease::Semver(release_semver) => {
                 let download_url = github_release_download_url(release_semver);
                 let release_dir = config.release_dir(&release_semver);
-                let download_url = if release_dir.join(".ok").exists() {
+                let download_url = if release_dir.exists() {
```

Comment:
No more ".ok" junk needed because the release dir now only exists once extraction has fully completed